### PR TITLE
Style existing pages + add gem logs page

### PR DESCRIPTION
**Pull request by ming-00**
- Swapped out headers for all relevant pages
 - Formatted friend cell component, friends pages, drop friend select page, gem logs skeleton
 - There is this really weird bug where I can't import my searchbar component into the add friends page without the searchbar breaking on all pages, I am shedding hair
 - Replace favicons

Will continue to add functionality but thought I should push the changes first since they span q a few files!

### CODE REVIEW

File: frontend/src/components/FriendCell.vue [hunk 1]
```diff
@@ -53,20 +53,31 @@ export default defineComponent({
...
+  margin-left: -8px;
+  width: 104%;
+}
+
+.van-cell {
+  border-radius: 100em;
+}
+
+.van-cell:hover {
+  background: @purple;
```
Comment (sebastiantoh) [inline frontend/src/components/FriendCell.vue]: should we use @primary-color instead?

File: frontend/src/router/index.ts [hunk 2]
```diff
@@ -72,6 +75,13 @@ const router = createRouter({
         isPublic: true
       }
     },
+    {
+      path: GEM_LOGS_ROUTE,
+      component: GemLogs,
+      meta: {
+        isPublic: true
+      }
```
Comment (sebastiantoh) [inline frontend/src/router/index.ts]: this should probably not be public? If my understanding is correct, public means accessible without being logged in

Comment (ianyong) [reply frontend/src/router/index.ts]: Yea, that's correct!

File: frontend/src/components/FriendCell.vue (hunk 1 above)
Comment (ming-00) [reply frontend/src/components/FriendCell.vue]: Okay! Good call

File: frontend/src/router/index.ts (hunk 2 above)
Comment (ming-00) [reply frontend/src/router/index.ts]: Ah ok! Tqtq

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**